app/Policies/MemberPolicy.php

```diff
@@ -32,7 +32,9 @@ public function view(User $user, Member $member)
     {
         return ($member->soort == "oud" && $user->hasCapability("members::oud::show"))
             || ($member->soort != "oud" && $user->hasCapability("members::info::show::basic"))
-            || ($this->ifSelf("members::info::show::self", $user, $member));
+            || ($this->ifSelf("members::info::show::self", $user, $member))
+            ||  $member->hasRole("counselor")
```

Comment:
This is not needed at this point. Discussed with Robin Klaassen. The data of the counselor is sent to the participants by snail mail / email